src/include/distributed/shard_rebalancer.h

```diff
@@ -77,7 +77,7 @@
 #define REBALANCE_PROGRESS_MOVING 1
 #define REBALANCE_PROGRESS_MOVED 2
 
-#define MAX_PARALLEL_TASKS_PER_NODE 128
+#define MAX_BACKGROUND_TASK_EXECUTORS_PER_NODE 128
```

Comment:
nit: Really this should be `MAX_MAX_BACKGROUND_TASK_EXECUTORS_PER_NODE` now. That seems even weirder. Let's just hardcode this number in the GUC definition function call, so we don't have to worry about naming this.

Reply:
Makes sense. FYI, we also use MAX_BG_TASK_EXECUTORS for max_background_task_executors :D

Reply:
Let's change that too then. Normally I'm no fan of combining feature changes with unrelated refactors, but this one seems small enough that it doesn't matter.

Reply:
On second thought, I will leave that as a note, and fix it in another PR. The MAX_BG_TASK_EXECUTORS constant is being used more than once. I don't want to mess up this PR too much.

Reply:
Sounds good. My assumption was that it was used one time only, just like `MAX_BACKGROUND_TASK_EXECUTORS_PER_NODE`